<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="q_btn_home">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img_btn/img_home.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="title">
     <string>수압 설정</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="q_rad_Level1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="text">
      <string>1단계</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="q_rad_Level2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="text">
      <string>2단계</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="q_rad_Level3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="text">
      <string>3단계</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="q_chkb_washonly">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="text">
      <string>닦기 전용</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="q_btn_stop">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>330</width>
      <height>210</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img_btn/img_stop.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="title">
     <string>위치 조절</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="q_btn_up">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="q_btn_down">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
